src/ring_theory/algebraic.lean

```diff
@@ -55,8 +55,12 @@ begin
   symmetry,
   -- TODO: add an `aeval`-specific version of `hom_eval₂`
   simp only [aeval_def],
-  convert hom_eval₂ p (algebra_map R S) ↑S.val ⟨x, hx⟩,
-  refl
+  have := hom_eval₂ p (algebra_map R S) (↑S.val : S →+* A) ⟨x, hx⟩,
```

Comment:
I'm not sure this is an improvement, it replaces one contentless line by six.  It might make more sense to add an `hom_aeval` lemma and use it here.

Reply:
how about this version?

Reply:
This is so much better now!